{{ cookiecutter.__package_name_kebab_case }}/.gitlab-ci.yml

```diff
@@ -62,7 +62,9 @@ Compute CI image hash:
         done
       fi
   rules:
-    - changes:
+    - if: $CI_PIPELINE_SOURCE != "merge_request_event"
+    - if: $CI_PIPELINE_SOURCE == "merge_request_event"
+      changes:
```

Comment:
FYI: `changes` will only work for MRs, so we need to explicitly say that we want to run `.docker` on the default branch as well so that the CI image is built on the first commit.

Reply:
Why the `!=` _and_ the `==`?

Reply:
You should read that as "run this code if this is not an MR" _or_ "run this code if this is an MR _and_ the following files are changed" [1].

[1] https://docs.gitlab.com/ee/ci/yaml/#rules